sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -839,6 +856,28 @@ export class ServiceBusAtomManagementClient extends ServiceClient {
     if (isUpdate) {
       webResource.headers.set("If-Match", "*");
     }
+
+    const forwardTo = (entityFields as InternalQueueOptions | InternalSubscriptionOptions)
+      .ForwardTo;
+
+    const forwardDeadLetterMessagesTo = (entityFields as
+      | InternalQueueOptions
+      | InternalSubscriptionOptions).ForwardDeadLetteredMessagesTo;
+
+    if (forwardTo && forwardTo.length > 0) {
+      webResource.headers.set(
+        "ServiceBusSupplementaryAuthorization",
+        (await this.sasTokenProvider.getToken(forwardTo)).token
+      );
+    }
+
+    if (forwardDeadLetterMessagesTo && forwardDeadLetterMessagesTo.length > 0) {
```

Comment:
If the length check is to ensure empty string is excluded, then it is not needed. `if (forwardDeadLetterMessagesTo)` should be enough

Reply:
updated